LEAF_Nexus/login/index.php

```diff
@@ -49,7 +51,13 @@ function getBaseDir() {
     This information system is provided for U.S. Government-authorized use only. Unauthorized or improper use of this system may result in disciplinary action, as well as civil and criminal penalties.<br /><br />
 
     <a href="<?php echo 'https://' . $_SERVER['SERVER_NAME'] . ':444' . getBaseDir() . 'auth_token/?' . htmlentities($_SERVER['QUERY_STRING']); ?>"><div class="buttonNorm" style="text-align: center">Login with your VA PIV card<img src="../../libs/dynicons/?img=go-next.svg&amp;w=32" alt="Icon for Login" title="Icon for Login" /></div></a><br />
-    <a href="<?php echo 'https://' . $_SERVER['SERVER_NAME'] . getBaseDir() . 'auth_domain/?' . htmlentities($_SERVER['QUERY_STRING']); ?>"><div class="buttonNorm" style="text-align: center">Login with Username and Password</div></a>
+    <a href="<?php if (Orgchart\Config::$leafSecure == false)
```

Comment:
The php code should encapsulate the entire <a> tag, otherwise a blank link is written to HTML.

Reply:
addressed in commit d47